components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/wsdl/SequenceGenerator.java

```diff
@@ -344,7 +345,20 @@ private static void saveApiSequences(APIIdentifier apiIdentifier, String inSeque
                         } else {
                             element = doc.createElementNS(null, parameterTreeNode);
                         }
+                        String xPathOfNode = "";
```

Comment:
Can we use a method in StringUtils to get this if possible?

Reply:
Added